codespell_lib/data/dictionary.txt

```diff
@@ -22410,6 +22434,7 @@ distructive->destructive
 distructor->destructor
 distructors->destructors
 distuingish->distinguish
+disturbative->distributive, disruptive
```

Comment:
`disturbative` is a word.

Reply:
TIL, but it's almost never used - much much much less frequently than `loath` ;) 

<img width="2334" alt="image" src="https://github.com/user-attachments/assets/c6ce2aa5-00a0-4aa3-9346-134d5148ca42">

But it don't mind taking it out, doesn't matter to me

Reply:
Then how about moving it to the _rare_ dictionary?

Reply:
Done!